vulcan/lib/client/jsx/api/vulcan.js

```diff
@@ -119,6 +120,13 @@ export const submit = (context) => {
     })
     .then((extractions) => {
       extractions.forEach((datum, index) => {
+        // We don't currently have accurate Content-Type headers
```

Comment:
:+1: 

I will get to work on content-type headers on the backend once I return to working fulltime on vulcan proper.  Thank you for being flexible.